witness/src/Dscp/Resource/Network.hs

```diff
@@ -83,8 +84,10 @@ instance WithNetLogging => AllocResource NetCliParams NetCliResources where
         buildComponentR "netcli" allocate release
       where
         allocate = do
+            ncPeers' <- liftIO $ Set.fromList <$> mapM mkZTNodeId (Set.toList ncPeers)
```

Comment:
Had to look in to recognise that this is a hand-made `mapM` on `Set`.

I'd make `mapMSet` function somewhere and move it later to some `*.Util`.